notebooks/kubeflow_pipelines/walkthrough/solutions/kfp_walkthrough_vertex_pytorch.ipynb

```diff
@@ -0,0 +1,1549 @@
+{
```

Comment:
<p>fix grammar: "In this lab you will develop a multi-class classification training application, package it as a docker image and run the application on Vertex AI."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/300/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks Sanjana, it seems the copying and pasting wasn't done correctly.</p>